src/commands/chat.ts

```
@@ -0,0 +1,67 @@
+  const prompt = interaction.options.getString('prompt', true);
+  const ctxId = interaction.guild
+    ? interaction.guild.id
+    : `dm:${interaction.user.id}`;
+
+  logIncoming(ctxId, interaction.user.username, prompt);
+
+  const chatContext = {
+    author: interaction.user,
+    content: prompt,
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    channel: interaction.channel!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)